### ISSUE TITLE
NVDA User Guide and touch commands

### ISSUE THREAD

**Issue by JamaicanUser** [nvaccess/nvda#5650]
In Section 5.4, which touches on object navigation, I have realised that there is a bit of an error or something confusing.
In the commands to move the navigator object to the next and previous object, the touch commands are incorrect. Single flick right and left do not perform the same action as NVDA+NUMPAD 6 and 4. They move in a flat representation of the hierarchy. 
I personally have assigned Input gestures to these touch commands (flat review), without a touch screen, and would hate to see them go, but feel there is a bit of confusion as to what single and two finger flick left and right do.

**Comment by leonardder** [comment]
I agree here.
@JCSTeh: since you are the initial of the author of the user guide section in question, could you please give your opinion?

**Comment by jcsteh** [comment]
Yes; flick left and flick right should be listed as separate commands. Historically, they behaved the same as NVDA+numpad4/6, but that changed some time ago and the documentation wasn't updated. P2 because of user confusion.

**Comment by bhavyashah** [comment]
@jcsteh I realize that the last activity on this ticket is just a little over a fortnight old, but still, a gentle ping to you and perhaps @josephsl to resolve the reported documentation issue whenever convenient.

**Comment by josephsl** [comment]
Hi, once I get more feedback on #7309 (touch keyboard), I’ll remind myself to implement suggestions on this issue. Thanks.

**Comment by Adriani90** [comment]
@josephsl I guess there is still a PR open regarding touch behavior. Right? Are there any further updates planned for userguide?

**Comment by josephsl** [comment]
Hi, please ping me regarding documentation changes requested throughout GitHub after 2019.1 beta 1 is released. Thanks.

**Comment by Adriani90** [comment]
Hey Joseph, since we are now at 2020.2 almost, are you planing to solve this in the documentation?

**Comment by josephsl** [comment]
Hi, thanks for the reminder. Now that #9682 is here, I’ll look into it. This is conditional based on how #11297 turns out in the end. Thanks.

**Comment by CyrilleB79** [comment]
At least since NVDA 2023.2 with #15053, this issue is fixed.
